database/javascript/2017/16174.yaml

```diff
@@ -0,0 +1,19 @@
+---
+cve: 2017-16174
+title: CVE in whispercast
+description: >
+    whispercast is a file server. whispercast is vulnerable to a directory traversal issue, giving an attacker access to the filesystem by placing "../" in the url.
+cvss_v2: 5.0
+references:
+    - https://github.com/JacksonGL/NPM-Vuln-PoC/blob/master/directory-traversal/whispercast
+    - https://nodesecurity.io/advisories/466
+
+affected:
+    - name: whispercast
+      version:
```

Comment:
https://nvd.nist.gov/vuln/detail/CVE-2017-16174?cpeVersion=2.2#vulnConfigurationsArea

Reply:
+1 please update the PR ;)